src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,61 @@
+import classNames from "classnames"
+import styles from "./ProgressBar.module.css"
+
+const MIN = 0
+const MAX = 100
+
+const getTransformStyle = (value?: number) => {
+  if (value || value === 0) {
+    const transform = value - 100
```

Comment:
I see some values (here and elsewhere) that are the same as the constants above. Would it make sense to utilize these throughout?
```suggestion
  if (value || value === MIN) {
    const transform = value - MAX
```